terra-jupyter-aou/Dockerfile

```diff
@@ -127,7 +130,7 @@ RUN echo "Sys.setenv(RETICULATE_PYTHON = '$(which python3)')" >> ~/.Rprofile
 
 # Install Notebook libraries as the user.
```

Comment:
nit: this comment is obsolete, most libraries are installed as root now in this Dockerfile